src/test/input_byte_size_test.cc

```diff
@@ -312,19 +313,20 @@ TEST_F(InputByteSizeTest, ValidInputByteSize)
 
 TEST_F(InputByteSizeTest, InputByteSizeMismatch)
 {
+  const char* model_name = "savedmodel_zero_1_float32";
   // Create an inference request
   FAIL_TEST_IF_ERR(
       TRITONSERVER_InferenceRequestNew(
-          &irequest_, server_, "pt_identity", -1 /* model_version */),
+          &irequest_, server_, model_name, -1 /* model_version */),
       "creating inference request");
   FAIL_TEST_IF_ERR(
       TRITONSERVER_InferenceRequestSetReleaseCallback(
           irequest_, InferRequestComplete, nullptr /* request_release_userp */),
       "setting request release callback");
 
   // Define input shape and data
-  std::vector<int64_t> shape{1, 8};
-  std::vector<float> input_data(10, 1);
+  std::vector<int64_t> shape{1, 16};
```

Comment:
Why changing these values?

Reply:
I don't remember. Might just want to be consistent with some other tests I saw. Either 8 or 16 is fine.